drape/glyph.hpp

```diff
@@ -37,12 +29,19 @@ struct GlyphImage
   SharedBufferManager::shared_buffer_ptr_t m_data;
 };
 
+using TGlyph = std::pair<int16_t /* fontIndex */, uint16_t /* glyphId */>;
```

Comment:
nit: Better to make a separate struct here. It is used multiple times like:
```
int16_t m_fontIndex;
uint16_t m_glyphId;
```
or
```
int16_t fontIndex, uint16_t glyphId
```